halyard/halyard-deploy/halyard-deploy.gradle

```diff
@@ -31,10 +31,10 @@ dependencies {
   implementation project(':halyard-backup')
 
   testImplementation 'org.spockframework:spock-core'
-  testImplementation 'org.springframework:spring-test'
   testImplementation 'org.mockito:mockito-core'
   testImplementation 'org.mockito:mockito-junit-jupiter'
   testImplementation 'org.junit.jupiter:junit-jupiter'
+  implementation 'org.springframework:spring-test'
```

Comment:
revert this change?  spring-test doesn't belong as an implementation dependency.